debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerMetrics.java

```diff
@@ -235,7 +235,7 @@ public int getBatchSize() {
 
     @Override
     public Integer getMillisecondToSleepBetweenMiningQuery() {
-        return millisecondToSleepBetweenMiningQuery.get();
+        return millisecondToSleepBetweenMiningQuery.intValue();
```

Comment:
I would suggest we actually return the value using `get()` here and set the return type as `long` as there is no need to return the value as `Integer` or even `int` when all the usages expect `long` anyway.
```suggestion
        return millisecondToSleepBetweenMiningQuery.get();
```

Reply:
I'll make the above change myself when I commit these changes; otherwise this LGTM.

Reply:
👍 I decided to go with intValue() assuming the override was in the parent class from debezium core. I see now it is in the jmx interface.